src/Service/ImageTypesProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfSylius\SyliusImportExportPlugin\Service;
+
+use FriendsOfSylius\SyliusImportExportPlugin\Repository\ProductImageRepositoryInterface;
+
+final class ImageTypesProvider implements ImageTypesProviderInterface
+{
+    public const IMAGES_PREFIX = 'Images_';
+
+    /** @var ProductImageRepositoryInterface */
+    private $productImageRepository;
+
+    public function __construct(ProductImageRepositoryInterface $productImageRepository)
+    {
+        $this->productImageRepository = $productImageRepository;
+    }
+
+    public function getProductImagesCodesList(?bool $prefix = true): array
+    {
+        $attrSlug = [];
+        $productImages = $this->productImageRepository->findTypes();
+        foreach ($productImages as $attr) {
```

Comment:
I would try to continue/break as soon as possible, so moving on in the code we really do things. It's harder to understand the deeper indented you have to read code.

```suggestion
        foreach ($productImages as $attr) {
            if (empty($attr['type'])) {
                continue;
            }
            
            $attrSlug[] = ($prefix ? self::IMAGES_PREFIX : '') . $attr['type']
        }
```

Reply:
Yeah, great :)